playbook/app/pb_kits/playbook/pb_filter/docs/_sort_only.jsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react'
 import { Filter } from '../../'
 
-const SortOnly = () => (
+const SortOnly = (props) => (
   <Filter
-      background={false}
+      {...props}
```

Comment:
I'm surprised this hasn't failed the lint check 🤔

Reply:
all of the .jsx files are indented 2 tabs instead of 1. Is that the proper format???